modules/programs/senpai.nix

```diff
@@ -61,13 +97,34 @@ in {
         <manvolnum>5</manvolnum></citerefentry>.
       '';
     };
+    extraConfig = mkOption {
+      type = types.attrs;
+      default = { };
+      description = ''
+        Options that should be appended to the senpai configuration file.
+      '';
+    };
```

Comment:
I am still not sure why we need this, instead of simply keeping the `freeformType`? This seems to be moving away from [RFC 42](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md).

Reply:
Unfamiliarity. Thanks for the link, fixed.